packages/graphql/tests/tck/tck-test-files/directives/node/node-label-union.test.ts

```diff
@@ -23,7 +23,7 @@ import { Neo4jGraphQL } from "../../../../../src";
 import { createJwtRequest } from "../../../../utils/create-jwt-request";
 import { formatCypher, translateQuery, formatParams } from "../../../utils/tck-test-utils";
 
-describe("Node directive with unions", () => {
+describe("!", () => {
```

Comment:
erm, I guess this is a typo :sweat_smile:

Reply:
Good catch!